src/components/LcTextarea/LcTextarea.stories.ts

```diff
@@ -0,0 +1,43 @@
+import LcTextarea from './LcTextarea'
+
+export default {
+  title: 'Example/LcTextarea',
+  component: LcTextarea,
+  argTypes: {
+    rules: { control: { type: 'select', options: [] } },
+  },
+}
+
+const Template = (args: any) => ({
+  components: { LcTextarea },
+  setup() {
+    return { args }
+  },
+  template: '<lc-textarea v-bind="args"/>',
+})
+
+export const Base = Template.bind({}) as any
+Base.args = {
+  label: 'Votre demande :',
+  name: 'content',
+  modelValue: '',
+  inputClass: 'w-full',
+}
+
+export const withCustomRows = Template.bind({}) as any
+withCustomRows.args = {
+  label: 'Votre demande :',
```

Comment:
Tu peut faire ça, idem pour celui en dessous ;) pour éviter de répété les key

```suggestion
withCustomRows.args = {
 ...Base.args,
  rows: 10,
}
```